<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QMainWindow" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_file_new"/>
    <addaction name="action_file_open"/>
    <addaction name="action_file_save"/>
    <addaction name="action_file_saveAs"/>
    <addaction name="separator"/>
    <addaction name="action_file_pageSetup"/>
    <addaction name="action_file_print"/>
    <addaction name="separator"/>
    <addaction name="action_file_exit"/>
   </widget>
   <widget class="QMenu" name="meu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_edit_undo"/>
    <addaction name="separator"/>
    <addaction name="action_edit_cut"/>
    <addaction name="action_edit_copy"/>
    <addaction name="action_edit_paste"/>
    <addaction name="action_edit_delete"/>
    <addaction name="separator"/>
    <addaction name="action_edit_selectAll"/>
    <addaction name="action_edit_timeDate"/>
    <addaction name="separator"/>
    <addaction name="action_edit_wordWrap"/>
   </widget>
   <widget class="QMenu" name="menu_search">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="action_search_find"/>
    <addaction name="action_search_findNext"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_help_helpTopics"/>
    <addaction name="separator"/>
    <addaction name="action_help_aboutNotepad"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="meu_edit"/>
   <addaction name="menu_search"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_file_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_file_open">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="action_file_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_file_saveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="action_file_pageSetup">
   <property name="text">
    <string>Page Setup...</string>
   </property>
  </action>
  <action name="action_file_print">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="action_file_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_edit_undo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="action_edit_cut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="action_edit_copy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="action_edit_paste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="action_edit_delete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_edit_selectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="action_edit_timeDate">
   <property name="text">
    <string>Time/Date</string>
   </property>
  </action>
  <action name="action_search_find">
   <property name="text">
    <string>Find...</string>
   </property>
  </action>
  <action name="action_search_findNext">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="action_help_helpTopics">
   <property name="text">
    <string>Help Topics</string>
   </property>
  </action>
  <action name="action_help_aboutNotepad">
   <property name="text">
    <string>About Notepad</string>
   </property>
  </action>
  <action name="action_edit_wordWrap">
   <property name="text">
    <string>Word Wrap</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
